README.md

```diff
@@ -113,3 +113,9 @@ Follow the [React Native Getting Started Guide](https://facebook.github.io/react
   - Start emulator
   - Start react packager: `$ react-native start`
   - Run in emulator: `$ react-native run-android`
+
+# Frequently Asked Questions
+
+**How is data handled on mobile devices after a user account is deactivated?**
```

Comment:
I think people need confirmation that the data is wiped when an account is deactivated.

However, it might be useful to have two FAQs. One is "What happens to the data on a mobile device when a user logs out?" and the other is "How is the data... ". But that can probably wait for when we transfer this info to https://docs.mattermost.com/overview/faq.html#mobile-applications after mattermost-mobile goes into production.